build/views/index.html

```diff
@@ -23,6 +23,10 @@
       <li><strong>Out of hours</strong>: <em class="out-of-hours">{{(index .SupportRota "out-of-hours").Member}}</em></li>
       <li><strong>Escalations</strong>: <em class="escalations">{{(index .SupportRota "escalations").Member}}</em></li>
     </ul>
+    <button onclick="app.showTeam('a')">Team A</button>
+    <button onclick="app.showTeam('b')">Team B</button>
+    <button onclick="app.showTeam('c')">Team C</button>
+    <button onclick="app.showAll()">Show all</button>
```

Comment:
This could be done in a way of:

in HTML:
```html
<button data-filter="a">Team A</button>
```

in JS:
```js
function filterTeam(e) {
  e.preventDefaults()

  const filter = $(e.target).attr('data-filter'); // a
  // if empty, show all.
  // find cards assigned to that team or neutral.
    // show all these cards
}

$(document)
  .on('click', '[data-filter]', filterTeam)
```

It's just a sample, and would need to be amended to TypeScript and the Application class.

Reply:
:-D yep, that sounds better. 

I would try to change it, I do not promise anything.